Interpreter_Constants.h

```diff
@@ -59,7 +59,11 @@ enum {
   MustBeBooleanSelector = 52,
   /*ST:  Tables */
   SpecialSelectorsPointer = 48,
-  CharacterTablePointer = 50
+  CharacterTablePointer = 50,
+  /**
+   * The ObjectMemory needs to know not to add the above pointers to the free pointer list on initialisation.
+   */
+  FirstUnusedObjectPointer = 52
```

Comment:
I want to track the end of this `enum` so that all `Interpreter`-defined objects have a refcount of one on start (except `NilPointer`, see below). If they're unused they'll get deallocated by the marking collector eventually, but my guess is that they shouldn't ever be unused because the `Interpreter` expects to be able to find them.